.github/workflows/deploy.yml

```diff
@@ -17,7 +17,7 @@ jobs:
   deploy:
     uses: nationalarchives/tdr-github-actions/.github/workflows/lambda_deploy.yml@main
     with:
-      lambda-name: export-status-update
+      lambda-name: export-failure-status
```

Comment:
I thought the new lambda name is: `export-status-lambda`?

Reply:
It is. This is an old PR; I forgot that it was still open 🤦